Bug 1128840 - [Stingray][Home] Let left margin of cardlist become custom...

rexboy7 opened the pull request:
...izable.

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/2406290?v=3" alt="rexboy7" width="50px" height="50px" /> Rex KM Lee (rexboy7)](https://github.com/rexboy7) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=3a1b253cf3d8)

rexboy7 committed:
Bug 1128840 - [Stingray][Home] Polish card list and animation style
               - Avoid trembling of scaling animation in edit mode
               - Add more left margin space before first card
               - Style fine-tuning (on animation time and some colors)

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/2406290?v=3" alt="rexboy7" width="50px" height="50px" /> Rex KM Lee (rexboy7)](https://github.com/rexboy7) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=856e2ebbd2cf)
